src/bd/primitives/Primitive.java

```diff
@@ -0,0 +1,84 @@
+package bd.primitives;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import com.oracle.truffle.api.dsl.NodeFactory;
+
+
+/**
+ * Annotation to be applied on node classes that are meant to be used as primitive operations.
+ * Thus, the node implements a built-in function of the interpreter, and might need to be
+ * represented as a language-level method/function.
```

Comment:
I wanted to communicate what a primitive is.
The node for an `+` might not only be an operator-like construct, but also a language level method. Since you can do `1 perform: #+ with: 3`, the `+` also needs to be a language-level method, not just something operator-like.

Does that make sense?
Anything that could be done to clarify the comment?